packages/recommend/src/__tests__/getFrequentlyBoughtTogether.test.ts

```diff
@@ -0,0 +1,83 @@
+import { TestSuite } from '../../../client-common/src/__tests__/TestSuite';
+
+const recommend = new TestSuite('recommend').recommend;
+
+function createMockedClient() {
+  const client = recommend('appId', 'apiKey');
+  // @ts-ignore read-only property
+  client.transporter = {
+    ...client.transporter,
+    read: jest.fn(),
+  };
+
+  return client;
+}
+
+describe('getFrequentlyBoughtTogether', () => {
+  test('builds the request', async () => {
+    const client = createMockedClient();
+
+    await client.getFrequentlyBoughtTogether(
+      {
+        indexName: 'products',
+        objectID: 'B018APC4LE',
+      },
+      {}
```

Comment:
is this a leftover from providing custom hosts?

Reply:
Nope, it's to make sure that the `requestOptions` are forwarded.